tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl

```diff
@@ -338,55 +338,63 @@
       </xsl:if>
     </xsl:template>
 
-    <xsl:template match="javaee:attribute">
+    <xsl:template match="jakartaee:attribute">
       <tr valign="top">
-        <td><xsl:apply-templates select="javaee:name"/></td>
+        <td><xsl:apply-templates select="jakartaee:name"/></td>
         <td>
           <xsl:choose>
-            <xsl:when test="javaee:required!=''">
-              <xsl:value-of select="javaee:required"/>
+            <xsl:when test="jakartaee:required!=''">
+              <xsl:value-of select="jakartaee:required"/>
             </xsl:when>
             <xsl:otherwise>false</xsl:otherwise>
           </xsl:choose>
         </td>
         <td>
           <xsl:choose>
-            <xsl:when test="javaee:rtexprvalue!=''">
-              <xsl:value-of select="javaee:rtexprvalue"/>
+            <xsl:when test="jakartaee:rtexprvalue!=''">
+              <xsl:value-of select="jakartaee:rtexprvalue"/>
             </xsl:when>
             <xsl:otherwise>false</xsl:otherwise>
           </xsl:choose>
         </td>        
         <td>
           <xsl:choose>
-            <xsl:when test="javaee:deferred-value">
+            <xsl:when test="jakartaee:deferred-value">
                 <xsl:choose>
+<<<<<<< HEAD
                     <xsl:when test="javaee:deferred-value/javaee:type">
```

Comment:
My guess is this is your issue with the error I'm getting locally, looks like a potential merge issue wasn't resolved.

Reply:
Fixed. Thanks for catching this.